src/Submission/TextInput.less

```diff
@@ -16,7 +17,10 @@
 }
 
 .text-input__input-description {
-  font-size: 1rem;
   display: inline-block;
   width: 750px;
-}
\ No newline at end of file
+}
+
+.ant-form-item {
```

Comment:
Possible to put this css on a wrapper div instead of `ant-form-item`? This could cause confusing behavior down the line if an ant form somewhere else in the website looks different because of this css.

Reply:
yeah good call! updated